apps/transmitmedia.cpp

```diff
@@ -161,6 +161,19 @@ void PrintSrtStats(int sid, const PerfMonType& mon)
     cerr << output.str() << std::flush;
 }
 
+void PrintSrtBandwidth(double mbpsBandwidth)
```

Comment:
Please make PrintSrtBandwidth() static

Reply:
I've added the `static` modifier. Should I do the same for `PrintSrtStats`? That's what I modeled this after...

Reply:
Thanks for pointing that out @boushley! I took the liberty of making the change directly on your branch since I was already looking.